bear.profile

```diff
@@ -45,4 +48,14 @@ function bear_form_node_form_alter(&$form, FormStateInterface $form_state) {
   if (isset($form['sticky'])) {
     $form['sticky']['#access'] = FALSE;
   }
+
+  $user = \Drupal::currentUser();
+  if (\Drupal::moduleHandler()->moduleExists('workbench_moderation') && isset($form['moderation_state'])) {
+    return;
+  }
+  if (!$user->hasPermission('view revisions')) {
```

Comment:
\Drupal::currentUser()->hasPermission('name of permission'); is a little more concise than doing it in 2 lines